recipe/meta.yaml

```diff
@@ -43,6 +44,7 @@ test:
     - pytest-mock
     - python
     - mercurial  # [py2k]
+    - mock # [py2k]
```

Comment:
Shouldn't we just fix `pytest-mock`? It's clearly broken right?

Reply:
Yeah, indeed. Wanted to see if this would in fact fix it...

Reply:
PR up: https://github.com/conda-forge/pytest-mock-feedstock/pull/13